Dockerfile

```diff
@@ -1,11 +1,24 @@
-FROM ubuntu:22.04
+FROM ubuntu:jammy
```

Comment:
why jammy instead of version ?

Reply:
its in the recommentations https://docs.google.com/document/d/1piO09xRbvwEwfru-oOgBpctA47x8kX_S2Fr7Y2ovHfs/edit#heading=h.g20qrwjts0x